packages/repl/src/repl.ts

```diff
@@ -90,6 +90,9 @@ const setupFileWatchers = async (repl: REPLServer) => {
 
 const runRepl = async (replOptions: REPL.ReplOptions) => {
   const repl = await initializeRepl(replOptions)
+  repl.on("exit", () => {
+    process.exit()
+  })
```

Comment:
Maybe make it shorter and more concise

```suggestion
  repl.on("exit", () => process.exit())
```